solve_triangle.py

```diff
@@ -0,0 +1,196 @@
+'''
+This script will help solve for an unknown side of a right triangle
+using the Pythagorean Theorem (PT) or basic trig functions.
+
+
+Recall that given a right triangle, the PT states that the sum of the
+square of the two legs of a triangle (the short sides) is equal to the
+square of its hypotenuse (the long side).
+
+The sin, cos, and tan functions may also be used to solve for unknown
+sides, and angles between them.
+
+A user can provide 2 of the three sides when calling main, and then run
+the script to determine what the unknown side is.
+
+Requirements:
+
+    The PT and trig funtions require all inputs to be positive
+    The PT and trig function assume the hypotenuse is the longest side -- by definition.
+    The code implementation requires that angles be provided in degrees.
+'''
+
+import argparse
+import math
+
+def apply_pythagorean_theorem(hypotenuse, leg_a, leg_b):
+    ''' Solve the the pythagorean theorem given 2 of 3 user inputs.
+
+    Pythagorean Theorem:
+
+        hypotenuse^2 = leg_a^2 + leg_b^2
+
+    Input:
+
+        Ordered list of values in the following order:
+
+        leg_a        length of one of the legs
```

Comment:
The order of the inputs listed here does not match the order in the function definition.